tests/importer/chain_ctx.go

```diff
@@ -0,0 +1,164 @@
+package importer
+
+import (
+	"math/big"
+
+	ethcmn "github.com/ethereum/go-ethereum/common"
+	ethcons "github.com/ethereum/go-ethereum/consensus"
+	ethstate "github.com/ethereum/go-ethereum/core/state"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+)
+
+// ChainContext implements Ethereum's core.ChainContext and consensus.Engine
+// interfaces. It is needed in order to apply and process Ethereum
+// transactions. There should only be a single implementation in Ethermint. For
+// the purposes of Ethermint, it should be support retrieving headers and
+// consensus parameters from  the current blockchain to be used during
+// transaction processing.
+//
+// NOTE: Ethermint will distribute the fees out to validators, so the structure
+// and functionality of this is a WIP and subject to change.
+type ChainContext struct {
```

Comment:
readded this and moved it into the importer directory because it's only used there